virtcontainers/tuntap_endpoint.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2018 Huawei Corporation
```

Comment:
- Is this copyright year still correct?
- Shouldn't we add a SoB for the original author in the commit?

Reply:
agreed....

@marcov FYI.

Reply:
This is a derived work of 
https://github.com/kata-containers/runtime/blob/master/virtcontainers/tap_endpoint.go

Can we leave `2018 Huawei` and add `2019 Intel...` ?

Reply:
Sounds reasonable to me.